31 document setup for backend (#106)
Avanade/emtech-distributed-data: docs/confidential-ledger-setup.md

`N.B Using the provided script will create and upload a cert for you, so you will not need to create your own cert`

> ### Steup - Register an Application

Reviewer: ```suggestion
### Setup - Register an Application
```

> In order to use the python SDK without using your Azure Default Credentials through VS code, in order to run the solution on a server for example we will need a client app id and a client secret.

Reviewer: ```suggestion
To run the application on a server, you'll need to register a client application ID and a client secret. In development, you can use the Python SDK in VS Code or another IDE, which will use your Default Azure Credentials, for example through `az login`.
```

Follow the instructions [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) to use your Azure subscription to generate the required information.

> Add teh following information to your .env file:

Reviewer: ```suggestion
Add the following information to your .env file:
```
